app/config/FrontendAppConfig.scala

```diff
@@ -59,6 +58,7 @@ trait AppConfig {
   val platformHost: String
   val directDebitInterruptUrl : String
   val btaHomeUrl: String
+  val accessibilityReportUrl: String
```

Comment:
This isn't being used anywhere. Can you remove it from here and all relevant places (the class below, ConfigKeys, mock config)